src/openapi.yaml

```diff
@@ -453,499 +1481,565 @@ paths:
+          type: object
+          additionalProperties:
+            type: string
+            title: kenmerk
+            description: Een waarde behorende bij de sleutel.
+            maxLength: 1000
+          description: Mapping van kenmerken (sleutel/waarde) van de notificatie.
+            De publicerende API specificeert de toegestane kenmerken.
       required:
-      - kanaal
+      - aanmaakdatum
+      - actie
       - hoofdObject
+      - kanaal
       - resource
       - resourceUrl
-      - actie
-      - aanmaakdatum
+    Notificatie:
       type: object
       properties:
         kanaal:
-          title: kanaal
+          type: string
           description: De naam van het kanaal (`KANAAL.naam`) waar het bericht op
             moet worden gepubliceerd.
-          type: string
           maxLength: 50
-          minLength: 1
         hoofdObject:
+          type: string
+          format: uri
           title: hoofdobject
           description: URL-referentie naar het hoofd object van de publicerende API
             die betrekking heeft op de `resource`.
-          type: string
-          format: uri
-          minLength: 1
         resource:
-          title: resource
-          description: De resourcenaam waar de notificatie over gaat.
           type: string
+          description: De resourcenaam waar de notificatie over gaat.
           maxLength: 100
-          minLength: 1
         resourceUrl:
-          title: resource-URL
-          description: URL-referentie naar de `resource` van de publicerende API.
           type: string
           format: uri
-          minLength: 1
+          description: URL-referentie naar de `resource` van de publicerende API.
         actie:
-          title: actie
+          type: string
           description: De actie die door de publicerende API is gedaan. De publicerende
             API specificeert de toegestane acties.
-          type: string
           maxLength: 100
-          minLength: 1
         aanmaakdatum:
-          title: aanmaakdatum
-          description: Datum en tijd waarop de actie heeft plaatsgevonden.
           type: string
           format: date-time
+          description: Datum en tijd waarop de actie heeft plaatsgevonden.
         kenmerken:
-          title: kenmerken
-          description: Mapping van kenmerken (sleutel/waarde) van de notificatie.
-            De publicerende API specificeert de toegestane kenmerken.
           type: object
           additionalProperties:
+            type: string
             title: kenmerk
             description: Een waarde behorende bij de sleutel.
-            type: string
             maxLength: 1000
+          description: Mapping van kenmerken (sleutel/waarde) van de notificatie.
+            De publicerende API specificeert de toegestane kenmerken.
+      required:
+      - aanmaakdatum
+      - actie
+      - hoofdObject
+      - kanaal
+      - resource
+      - resourceUrl
+    PatchedAbonnement:
```

Comment:
Not sure why this resource is not named `Abonnement`? :thinking:

Reply:
`drf-spectacular` creates a separate schema for Patch endpoints - it makes all fields not required there. So now there are both `Abonnement` and `PatchedAbonnement` components in the OAS